mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/ReIndexerImplTest.java

```diff
@@ -154,4 +159,89 @@ void messageReIndexShouldBeWellPerformed() throws Exception {
         assertThat(messageCaptor.getValue()).matches(message -> message.getMailboxId().equals(mailboxId)
             && message.getUid().equals(createdMessage.getUid()));
     }
+
+    @Test
+    void messageReIndexUsingMailboxIdShouldBeWellPerformed() throws Exception {
```

Comment:
I think that you can replace "WellPerformed" with what you actually asserts : `shouldCallMessageSearchIndex`